cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -230,6 +231,14 @@ handleSimpleNode p trace nodeTracers nc onKernel = do
   publicRootsVar <- newTVarIO publicRoots
   useLedgerVar <- newTVarIO (useLedgerAfterSlot nt)
 
+  _ <- Signals.installHandler
```

Comment:
@bolt12 Can we add to the docs (`doc/getting-started/understanding-config-files.md`) under the heading `#### The topology.json file` how to update the topology on the fly?

Reply:
Also this is being merged into `bolt12/master-diffusion-arguments `. Was that your intention?

Reply:
It was! Over at `ouroboros-network` I also have different PRs that build one top of the other to ease reviewing. Here's the `bolt12/master-diffusion-arguments` PR: #2647

So it should be #2635 -> #2647 -> #2652